pkg/controller/batchrelease/workloads/cloneset_control_plane.go

```diff
@@ -112,6 +112,22 @@ func (c *CloneSetRolloutController) PrepareBeforeProgress() (bool, error) {
 	return true, nil
 }
 
+func (c *CloneSetRolloutController) PrepareOneBatch() (bool, error) {
+	if err := c.fetchCloneSet(); err != nil {
+		return false, err
+	}
+
+	if !c.clone.Spec.UpdateStrategy.Paused {
+		return true, nil
+	}
+
+	err := c.patchCloneSetPaused(c.clone, false)
```

Comment:
do we really need to pause the cloneset, if the user change workload podtemplate, webhook will pause the workload anyway